autograder/api/submission/fetchattempts.py

```diff
@@ -0,0 +1,24 @@
+import autograder.api.common
+import autograder.api.config
+
+API_ENDPOINT = 'submission/fetch/attempts'
+API_PARAMS = [
+    autograder.api.config.PARAM_COURSE_ID,
+    autograder.api.config.PARAM_USER_EMAIL,
+    autograder.api.config.PARAM_USER_PASS,
+    autograder.api.config.PARAM_ASSIGNMENT_ID,
+
+    autograder.api.config.PARAM_TARGET_EMAIL_OR_SELF,
+]
+
+DESCRIPTION = 'Get all attempts made by a user along with all grading information.'
```

Comment:
s/attempts/submission attempts/

I have no strong reasoning for this, I just think it will look better in the command's USAGE.